doc/userguide/rules/payload-keywords.rst

```diff
@@ -749,6 +749,18 @@ the calculated entropy value with the buffer on which the value was computed::
         }
       }
 
+When ``logging.entropy.make-unique`` is enabled, the format is modified such
+that the signature id of the rule producing the entropy value and the occurrence
+value is included::
+
+     "metadata": {
+        "entropy": [
+          {
+            "sid:39133;buffer:file_data;instance:1": 4.265743301617466
+          }
```

Comment:
As they keys here are variable, I think we should consider turning them into values:
```json
"entropy": [
  {"key": "sid:39133;buffer:file_data;instance:1", "value": 4.265743301617466}
]
```
to avoid blowing up the indices in the database which might automatically index all keys.

Reply:
I think you are right, but I guess we should then also get rid of the key as a big concatenated string and instead break it out into a proper object.

Reply:
Like this?

```
"entropy": [
  { "sid": 39133, "buffer": "file_data", "instance": 1, "value": 4.265743301617466}
]
```

Reply:
What I like about `{"key": "sid:39133;buffer:file_data;instance:1", "value": 4.265743301617466}` is that the key is free-form, no schema needed. Would there be a use case for filtering on any specific fields within the key?

If the desire is is break down the key into an object, then I think I `key` object where `additionalProperties: true` is used to keep it free-form. Or are we pretty set on this?